### PR TITLE
Show icon only option for the clone counter addon

### PR DESCRIPTION
**Pull request by OregSamSas**
Resolves #{ϕ}

### Changes

 I've added a new option for the addon `clones` which make the monitor display just the icon and the clone number, without the clones text. 

### Reason for changes

Because RIMOPA (https://github.com/RIMOPA) requested it on discord [Discord comment](https://discord.com/channels/806602307750985799/826250884279173162/972687955011465227), and I support the idea, and also have an idea how to implement it, so i made it (**They own the idea I only implemented it**), it's a small change, but very useful for small screens.

### Tests

 I tested it on Chrome and on Waterfox (Firefox based) on this project: https://scratch.mit.edu/projects/659873974/editor/, and it worked fine. 

### Further reading/History/Related stuff

 You can have a look at the previous (outdated) pull request of mine featuring the same changes here: #4549

### CODE REVIEW

**Comment by OregSamSas** [comment]
I'm wondering if someone could tell me what did I do wrong other than everything? Because I'd need a more precise answer to fix this issue that sounds very **serious**: https://github.com/ScratchAddons/ScratchAddons/runs/7200877690 (It says: ***detached head...***)

**Comment by RIMOPA** [comment]
In the branch, go to the actions tab and enable them

**Comment by Norbiros** [comment]
> In this branch, go to the actions tab and enable them

Also you have commits on master branch, this can be problem too

**Comment by WorldLanguages** [comment]
@OregSamSas Don't worry about the Prettier action

File: addons/clones/userscript.js [hunk 1]
```diff
@@ -35,14 +37,19 @@ export default async function ({ addon, global, console, msg }) {
 
   const cache = Array(301)
     .fill()
+    //Shows just the clone count if the setting is enabled
```
Comment (apple502j) [inline addons/clones/userscript.js:40]:
```suggestion
```

**Review by apple502j — APPROVED** [review]

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**